napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -180,6 +193,18 @@ def _zoom_out(viewer: Viewer):
         callback=_zoom_out,
         keybindings=[StandardKeyBinding.ZoomOut],
     ),
+    Action(
+        id='napari.viewer.dims.ndisplay',
```

Comment:
```suggestion
        id='napari.window.view.toggle_ndisplay',
```
I played with it and I prefer going for consistency. 
We have an open issue to track renaming, so we can get all of them if we don't like 'em.
It's not clear to me the API path is better than the menu path for the command palette, which is the only place these ids are seen at the moment.